waku/v2/node/wakunode2.nim

```diff
@@ -392,9 +392,11 @@ proc info*(node: WakuNode): WakuInfo =
   ## Status: Implemented.
   ##
 
-  # TODO Generalize this for other type of multiaddresses
   let peerInfo = node.peerInfo
-  let listenStr = $peerInfo.addrs[^1] & "/p2p/" & $peerInfo.peerId
+  var listenStr = ""
+  for address in node.announcedAddresses:
+    var fulladdr = "[" & $address & "/p2p/" & $peerInfo.peerId & "]" 
```

Comment:
Don't feel too strongly about this, but it seems a bit more idiomatic to me if the output was something like
> "[`listenstrAddr1`, `listenstrAddr2`, ...]"

rather than

> "[`listenstrAddr1`][`listenstrAddr2`]..."

I think to attain that you can create a `seq` of `listenstr`s and just use the canonical `$` stringify for the sequence.

Reply:
changed format in 08fd37dabd54bcd0286890d5e610a8688ac6af2a